types/puppeteer/index.d.ts

```diff
@@ -2249,3 +2308,50 @@ export function executablePath(): string;
 export function launch(options?: LaunchOptions): Promise<Browser>;
 /** This methods attaches Puppeteer to an existing Chromium instance. */
 export function createBrowserFetcher(options?: FetcherOptions): BrowserFetcher;
+/**
+ * Predefigned puppeter devices
+ * 
+ * see DeviceDescriptors.js for full list
+ */
+export const devices: {
+  [key: string]: Device;
+};
+export const error: Error | undefined;
```

Comment:
This doesn't appear to exist?

Reply:
added in v1.15.0
see https://github.com/GoogleChrome/puppeteer/releases

Reply:
Maybe I can add an enum of the existing devices  😁